tfe.go

```diff
@@ -942,7 +942,7 @@ func checkResponseCode(r *http.Response) error {
 				return ErrWorkspaceLockedCannotDelete
 			}
 			if errorPayloadContains(errs, "being processed") {
-				return ErrWorkspaceStillProcessing
+				return ErrWorkspaceStillProcessing // as of 2024-03-04, the error message coming from atlas is "Latest workspace state is being processed to discover resources, please try again later"
```

Comment:
This comment would be useful to include in the godocs; you can move this comment above the definition for `ErrWorkspaceStillProcessing`. I would also change `atlas` to `API` since that might lead to some confusion.